src/tasks-api/BusinessLayer/ILearningTaskService.cs

```diff
@@ -0,0 +1,10 @@
+﻿using EKids.Chatbot.Tasks.DataAccessLayer.Entities;
+
+namespace EKids.Chatbot.Tasks.BusinessLayer;
```

Comment:
It is necessary to select layers for different projects

Reply:
Moved layers to folders with separate projects